test/utils/EasyPosm.sol

```diff
@@ -37,27 +37,47 @@ library EasyPosm {
         uint256 deadline,
         bytes memory hookData
     ) internal returns (uint256 tokenId, BalanceDelta delta) {
-        (Currency currency0, Currency currency1) = (poolKey.currency0, poolKey.currency1);
+        (Currency currency0, Currency currency1) = (
+            poolKey.currency0,
+            poolKey.currency1
+        );
```

Comment:
is this from forge fmt linting?

Reply:
This is indeed for forge linting but turns out it results in ever so slightly different formatting based on config. I've added a tiny config file to force use repo linting config.